collector/stat_activity.go

```diff
@@ -32,9 +32,10 @@ USING (datname, state) /*postgres_exporter*/`
 
 	// Oldest transaction timestamp
 	statActivityCollectorXactQuery = `
-SELECT min(xact_start)
+SELECT COALESCE(min(xact_start), current_timestamp) AS xact_start
   FROM pg_stat_activity
-  WHERE state IN ('idle in transaction', 'active') /*postgres_exporter*/`
+  WHERE state IN ('idle in transaction', 'active')
+  AND backend_xid IS NOT NULL /*postgres_exporter*/`
```

Comment:
Deserves a comment here explaining why we're ignoring this, otherwise the reasoning will be lost to the wind

Reply:
Check line 34 and 35 https://github.com/rnaveiras/postgres_exporter/pull/9/commits/40d579fd9d6a560210d1ad6f6aec9e8ac5904c45#diff-b5933fa2aa3a1f3cec87c50e14818786R34 I just add the comment there

Reply:
Silly me, all good 👍